### ISSUE TITLE
Fail to generate code (kotlin) for rpc method returning google.protobuf.Empty

### ISSUE THREAD

**Issue by slaaap** [square/wire#2478]
Hello,

I was trying to generate server side code using wire-gradle-plugin (with gRPC Server compatible mode enabled):
```kotlin
wire {
    sourcePath {
        srcDir("${project.rootDir}/api/src/main/proto")
    }
    kotlin {
        rpcRole = "server"
        rpcCallStyle = "suspending"
        grpcServerCompatible = true
    }
}
```
My proto file content is as following:
```proto
syntax = "proto3";
import "google/protobuf/empty.proto";

service MyService {
  rpc doSomething(google.protobuf.Empty) returns (google.protobuf.Empty);
}
```
Because `google.protobuf.Empty` type is mapped to `kotlin.Unit,` it generates following marshaller in `MyServiceWireGrpc.kt` in this case:
```kotlin
    public class EmptyMarshaller : WireMethodMarshaller<Unit> {
      public override fun stream(`value`: Unit): InputStream =
          Unit.ADAPTER.encode(value).inputStream()

      public override fun marshalledClass(): Class<Unit> = Unit::class.java

      public override fun parse(stream: InputStream): Unit = Unit.ADAPTER.decode(stream)
    }
```
Which cannot be compiled because `Unit` class is not generated and does not have custom `ADAPTER` inside
Seems that some specific logic during generation /or custom predefined marshaller is needed to process `Empty` as return/request type

Please let me know if I miss something and/or any additional details are needed

**Comment by oldergod** [comment]
It's a grpc-server specific issue.

@jvmakine what do you think?

**Comment by jvmakine** [comment]
Hi, thank you for reporting this! This should fix the compilation issue: https://github.com/square/wire/pull/2506

**Comment by oldergod** [comment]
Fixed with #2506
